src/AaveV2_Eth_CRVLTUpdate_20230806/AaveV2_Eth_CRVLTUpdate_20230806.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV2PayloadEthereum} from 'aave-helpers/v2-config-engine/AaveV2PayloadEthereum.sol';
+import {AaveV2Ethereum, AaveV2EthereumAssets} from 'aave-address-book/AaveV2Ethereum.sol';
+
+
+/**
+ * @title Chaos Labs Risk Parameter Updates - CRV Aave V2 Ethereum
+ * @author @omeragoldberg - ChaosLabsInc
+ * - Snapshot: https://snapshot.org/\#/aave.eth/proposal/0x8b992ee05d9e87ef0dab2cb7178c24f7b4b6f5d79561ad33298550b3c8d9fe89
+ * - Discussion: https://snapshot.org/\#/aave.eth/proposal/0x8b992ee05d9e87ef0dab2cb7178c24f7b4b6f5d79561ad33298550b3c8d9fe89
```

Comment:
this should be forum link

Reply:
@sendra updated to the forum link, thanks!